tools/clang/tools/dxa/dxa.cpp

```diff
@@ -281,16 +286,32 @@ void DxaContext::ListParts() {
 using namespace hlsl::options;
 
 int __cdecl main(int argc, _In_reads_z_(argc) char **argv) {
+  if (llvm::sys::fs::SetupPerThreadFileSystem())
+    return 1;
```

Comment:
should there be an error message here?

Reply:
The way stdout/stderr is hooked in, if this fails, we will crash if we try to print anything.  This really should not ever fail.